test/fail_compilation/fail_scope.d

```diff
@@ -11,7 +11,7 @@ fail_compilation/fail_scope.d(68):     `fail_scope.foo8(return ref int x)`
 fail_compilation/fail_scope.d(81): Error: returning `& string` escapes a reference to local variable `string`
 fail_compilation/fail_scope.d(91): Error: returning `cast(int[])a` escapes a reference to local variable `a`
 fail_compilation/fail_scope.d(99): Error: returning `cast(int[])a` escapes a reference to local variable `a`
-fail_compilation/fail_scope.d(107): Error: escaping reference to outer local variable `x`
+fail_compilation/fail_scope.d(107): Deprecation: escaping reference to outer local variable `x`
```

Comment:
?

Reply:
`-preview=dip25` makes it an `Error` (old behavior), but when I enabled it by default, I obviously made it a deprecation.
Even with `-de` it would show up as `Deprecation` (but behave like an error).